portality/tasks/article_bulk_delete.py

```diff
@@ -0,0 +1,128 @@
+from copy import deepcopy
+import json
+
+from flask_login import current_user
+
+from portality import models
+
+from portality.tasks.redis_huey import main_queue
+from portality.decorators import write_required
+
+from portality.background import AdminBackgroundTask, BackgroundApi, BackgroundException
+
+
+def article_bulk_delete_manage(selection_query, dry_run=True):
+
+    if dry_run:
+        ArticleBulkDeleteBackgroundTask.check_admin_privilege(current_user.id)
+        return ArticleBulkDeleteBackgroundTask.estimate_delete_counts(selection_query)
+
+    ids = ArticleBulkDeleteBackgroundTask.resolve_selection_query(selection_query)
+    job = ArticleBulkDeleteBackgroundTask.prepare(
+        current_user.id,
+        selection_query=selection_query,
+        ids=ids
+    )
+    ArticleBulkDeleteBackgroundTask.submit(job)
+
+    return True
```

Comment:
Shouldn't this also return the delete count estimate, so there's a consistent return type?  Does the UI also need the delete count to be returned once the job has been successfully delivered?